test/framework/src/main/java/org/elasticsearch/index/mapper/MapperServiceTestCase.java

```diff
@@ -339,6 +340,8 @@ protected final XContentBuilder runtimeFieldMapping(CheckedConsumer<XContentBuil
         });
     }
 
+    protected static FieldDataContext EMPTY_CONTEXT = new FieldDataContext("test", null);
```

Comment:
Is it better to use `staticContext` here?

I wonder about the name `static`, btw. The name gets me mixed up with the `static` keyword.

Reply:
Maybe `empty` is a better name than `static` for the method as well?

Reply:
I've renamed to `noScriptContext`, as the primary usecase seems to be places where we don't support runtime scripts in fielddata.